TheraLang.Tests/Services/UserManagmentServiceTests.cs

```diff
@@ -1,10 +1,230 @@
-﻿using System;
+﻿using Common.Helpers.PasswordHelper;
+using FluentAssertions;
+using MockQueryable.Moq;
+using Moq;
+using System;
 using System.Collections.Generic;
-using System.Text;
+using System.Linq;
+using System.Linq.Expressions;
+using System.Threading.Tasks;
+using TheraLang.BLL.Interfaces;
+using TheraLang.BLL.Services;
+using TheraLang.DAL.Entities;
+using TheraLang.DAL.Repository;
+using TheraLang.DAL.UnitOfWork;
+using TheraLang.Tests.DataBuilders.ResourcesBuilders;
+using Xunit;
 
 namespace TheraLang.Tests.Services
 {
-    class UserManagmentServiceTestscs
+    public class UserManagmentServiceTests
```

Comment:
we can add base class with all the setup and ingerit this object

Reply:
done